openlibrary/templates/type/edition/view.html

```diff
@@ -339,13 +339,13 @@ <h3>
           <hr>  
           $if edition.first_sentence:
             <div>
-            <h3 class="collapse">$_("First Sentence")</h3>
```

Comment:
`collapse` class is only styled on the homepage static/css/components/home.less
Has no impact on JS. Likely an old artifact from another time.